releasenotes/notes/deprecate-process_config.enabled-89e9ab7b0193cc04.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Added `process_config.process_collection.enabled` and `process_config.container_collection.enabled` settings.
+    Use these new settings instead of `process_config.enabled` to control continer and process collection in the process agent.
```

Comment:
```suggestion
    Use these new settings instead of `process_config.enabled` to control container and process collection in the process agent.

```